csharp/ql/test/shared/FlowSummaries.qll

```diff
@@ -0,0 +1,38 @@
+import semmle.code.csharp.dataflow.FlowSummary
+import semmle.code.csharp.dataflow.internal.FlowSummaryImpl::Private::TestOutput
+
+abstract class IncludeSummarizedCallable extends RelevantSummarizedCallable {
+  /** Gets the qualified parameter types of this callable as a comma-separated string */
```

Comment:
All ql doc comments should end with a full stop `.`

Reply:
Will fix (and also look at the other comments). Is there a documented coding standard?

Reply:
Yes, we have https://github.com/github/codeql/blob/master/docs/qldoc-style-guide.md.

Reply:
@hvitved : Thank you! Small note: If the provided link is book-marked somewhere on your machine, then consider to change the link to point to "main" instead of "master" otherwise you won't see any new exiting changes to the coding standard 😄